### DISCUSSION TITLE
Improved Android TV Support

### DISCUSSION THREAD

**Discussion by SecularSteve** [breezy-weather/breezy-weather#565] [Ideas]
Tested out a sideloaded version of this application on a Chromecast with Google TV. It looks fairly nice, and seems to be performing with no issues.

However, the up/down scrolling does not work on the main weather screen when I've tried using the up/down buttons on my remote, and the tab won't switch over in the Settings/Preferences menu to allow me to access any of the other menus with the highlight.

To sum it up in a single sentence: "Needs better remote controller support."

My point is, the "Android TV Stick/Box" field is kind of barren of Open Source alternatives. And Breezy has the potential to take advantage of that. Especially on TV devices with shoddy support for Google Services.

In order to get started, the App needs a Banner Icon for Android TV Launchers. If you're up for my suggestion, I could make it free of charge.

**Comment by papjul** [comment]
I don't have a TV, feel free to open an issue > Idea to add support for TV, then you can make the relevant fixes through a pull request. Thank you!

**Comment by papjul** [comment]
#566
